pkg/ovs/openflow/cmd_action.go

```diff
@@ -1,219 +0,0 @@
-// Copyright 2019 Antrea Authors
```

Comment:
I assume you already checked all new code after removing the cmd implementation to make sure the design is clean and no more include unnecessary logic introduced by the cmd implementation?

Reply:
Actually, I am not sure if we need to leave the method `String` in `Flow`. This method is not used, but the go binding needs to have some extra fields and processes to support it.

Reply:
I thought the String() is used by CLI - no? Guess you should check with @weiqiangt.

Anyway, I saw you already removed it. We could add it back with the CLI implementation later if needed.

Reply:
Sure. I have checked with @weiqiangt , currently CLI doesn't require to print Openflow entry.